dotnet/src/SemanticKernel.Abstractions/Models/PromptModel.cs

```diff
@@ -0,0 +1,80 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Collections.Generic;
+using System.Text.Json.Serialization;
+using Microsoft.SemanticKernel.AI;
+
+namespace Microsoft.SemanticKernel.Models;
+
+/// <summary>
+/// Prompt model files.
+/// </summary>
+public sealed class PromptModel
+{
+    /// <summary>
+    /// Name of the semantic function.
```

Comment:
Which name are we going forward?
```suggestion
    /// Name of the prompt function.
```